README.md

```diff
@@ -32,9 +32,9 @@ $ cd ./clara-platform-python-client
 $ python3 -m pip install .
 ```
 
-#### Installing directly from [Pypi](https://pypi.org/project/claraclient/)
+#### Installing directly from [Pypi](https://pypi.org/project/nvidia-clara/)
 ```
-$ python3 -m pip install claraclient
+$ python3 -m pip install nvidia-clara
```

Comment:
I don't think `nvidia-clara` is a great name for our package -- which Clara? Deploy? Train? Guardian? other?